javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,133 @@
+let myName = "";
+const toDoList = [];
+
+function getReply(command) 
+{
+    let x = command.toLowerCase();
+    const start = x.indexOf("is");
+    const name = x.substring(start + 2).trim();
```

Comment:
These two should be inside the "my name is" if-statement, since that's where they're used.